### PR TITLE
 internal/function: implement uast and uast_xpath functions

### LINKED ISSUE #193: Implements UDF that returns the UAST from a file
UDF that internally uses bblfsh. It should be like:

- `uast(content:BLOB, [lang:TEXT],[xpath:TEXT]) BLOB`

Also we will need another UDF to query again an uast

- `uast_xpath(uast:BLOB,xpath:TEXT) BLOB`

### PR DESCRIPTION
**Pull request by erizocosmico**
Closes #193

### CODE REVIEW

**Comment by erizocosmico** [comment]
Tests fixed, required some work done on the CI for bblfsh/client-go to work.

File: session.go [hunk 1]
```diff
@@ -11,20 +16,89 @@ import (
...
+	for {
+		if attempts > bblfshMaxAttempts {
+			return nil, ErrBblfshConnection.New()
+		}
+
+		switch s.bblfshClient.GetState() {
+		case connectivity.Ready, connectivity.Idle:
+			return s.bblfshClient, nil
+		case connectivity.Connecting:
+			time.Sleep(100 * time.Millisecond)
```
Comment (jfontan) [inline session.go]: Shouldn't we reset or decrease attempt counter? If it's connection at attempt 10 it will exit the next loop.

Comment (erizocosmico) [reply session.go]: That could lead to it being in an infinite loop if it's connecting, then fails, then connecting, then fails. We could set a maximum total retries and reset only a local counter, tho

**Comment by erizocosmico** [comment]
Done @jfontan

**Comment by erizocosmico** [comment]
Rebased

**Comment by erizocosmico** [comment]
Rebased and CI fixed.

**Comment by erizocosmico** [comment]
@ajnavarro integration test added

**Pull request merged**